picoquic/logger.c

```diff
@@ -658,9 +651,6 @@ size_t picoquic_log_generic_close_frame(FILE* F, uint8_t* bytes, size_t bytes_ma
         }
         if (ftype != picoquic_frame_type_connection_close || lf != 0) {
             l1 = picoquic_varint_decode(bytes + byte_index, bytes_max - byte_index, &string_length);
-            if (l1 == 0) {
-                required = byte_index + picoquic_varint_skip(bytes + byte_index);
-            }
         }
```

Comment:
My code seems wrong, but I am not sure that this is the right fix. Let me take a further look.

Reply:
We should keep this test, remove the unused reference to "required", and add the same kind of code as in the block above:
```
    if (l1 == 0) {
        byte_index = bytes_max;
    } else {
        byte_index += l1;
    }
```
The "required" variable was originally used in an error message, "only x bytes available when y bytes are required". Instead of removing the references, I would consider adding back the error message.

Reply:
Sorry, wrong suggestion. Your fixes here are fine.